src/tests/eina/eina_test_file.c

```diff
@@ -527,7 +527,11 @@ static const struct {
    const char *test;
    const char *result;
 } sanitize[] = {
+#if defined _WIN32
+  { "C:/home/mydir/../myfile", "C:/home/myfile" }
```

Comment:
Wouldn't be better to use `\\` instead of `/`?

Reply:
oh, good point, this change makes the `eina_test_file_path` pass through.

Reply:
@walac do you know why using `/` is working?

Reply:
Windows supports `/` as a path separator. I am working on more test cases for this.

Reply:
Indeed but I don't see as an option of choice for win32 api as its functions will only return `\\` as path separators.

Reply:
I am not sure I am following. In the updated patch I added tests for the `\` seperator cases.

Reply:
yeah, the updated is ok to me.